tests/scripts/config.source

```diff
@@ -1,26 +1,5 @@
-# Provide values from config
-settings="$(dirname "$(realpath "$0")")/config/pulp/settings.toml"
```

Comment:
@daviddavis the riddance of this line should help you to unblock your test subdirectory story.